.github/workflows/generate_pr_gold.yml

```diff
@@ -25,6 +25,15 @@ jobs:
       with:
           just-version: 1.26.0
 
+    - name: Take care of versioning
+      run: | 
+        source ${GITHUB_WORKSPACE}/.github/actions/common.sh
+        start_group Update Gold Label
```

Comment:
The `start_group` and `end_group` are helpful for separating messages within a single job step. Since this job step is only changing the version, I don't think nesting the messages within a group is necessary.

```suggestion
```